src/components/search/detailed/DetailedSearchResults.js

```diff
@@ -86,16 +87,18 @@ function DetailedSearchResults(props) {
 
     const dataTabIcon =
         selectedTab === SEARCH_RESULTS_TABS.data
-            ? "/data_selected.png"
-            : "/icon-data.png";
+            ? "/data.png"
+            : "/data-blue.png";
     const appsTabIcon =
-        selectedTab === SEARCH_RESULTS_TABS.apps
-            ? "/apps_selected.png"
-            : "/icon-apps.png";
+        selectedTab === SEARCH_RESULTS_TABS.apps ? (
+            <AppsIcon color="primary" />
+        ) : (
+            <AppsIcon />
+        );
```

Comment:
I thought `color="primary"` would make the icon blue, since our theme's primary color is CyVerse blue, but the other tab icons are using blue when the tab is not selected 🤔

Reply:
Yup..That why I have requested `info` colored icons from Mariah. Will be updated once I get it to match the current look in `prod` Sonora.